index.html

```diff
@@ -917,7 +918,7 @@
           Interface <a><code>PresentationRequest</code></a>
         </h3>
         <pre class="idl">
-          [Constructor(DOMString url)]
+          [Constructor(DOMString[] urls)]
```

Comment:
Shouldn't we keep a `Constructor(DOMString url)` around as well that allows devs to pass only one presentation URL without having to wrap it in a one-item array? I realize that fallback URLs are going to be needed for some time, but I would keep the longer-term goal of "one presentation URL that works everywhere" in the API design (similar to the possibility to give only one "src" to a video tag)